Parse-Dashboard/app.js

```diff
@@ -171,20 +171,27 @@ module.exports = function(config, options) {
       if (!users || (req.user && req.user.isAuthenticated)) {
         return res.redirect(`${mountPath}apps`);
       }
-
       let errors = req.flash('error');
       if (errors && errors.length) {
         errors = `<div id="login_errors" style="display: none;">
           ${errors.join(' ')}
         </div>`
       }
+      const customBrandIcon = config.customBrandIcon;
+      const customBrandColorPrimary = config.customBrandColorPrimary;
       res.send(`<!DOCTYPE html>
         <head>
           <link rel="shortcut icon" type="image/x-icon" href="${mountPath}favicon.ico" />
           <base href="${mountPath}"/>
           <script>
             PARSE_DASHBOARD_PATH = "${mountPath}";
+            ${customBrandIcon && 'CUSTOM_BRAND_ICON = "' + customBrandIcon + '";'}
```

Comment:
Is there a different way to pass this down to `SidebarHeader.react.js` than storing that on global scope?
Check `ParseApp.js`, somehow config variables are reaching that class.

Reply:
This `customBrandIcon` is the global icon for the whole `ParseDashboard` instance but `ParseApp.js` is the config of particular `ParseServer` instance which is available in `ParseDashboard` so it does not make sense to put `customBrandIcon` to it.

`customBrandIcon` should be part of some `ParseDashboardConfig.js` but this is much bigger refactor of the React app, I suggest to integrate this change with simple exposing config from server-side Express app to the client-side React app via global scope.

And, wrap it in the future to some general configuration file, or maybe to this endpoint which is the bridge between Express and React:
`app.get('/parse-dashboard-config.json', function(req, res) {`